### PR TITLE
Improve redundant variable name

### PR DESCRIPTION
**Pull request by pickfire**
## Description

Make documentation code more pythonic.

### Types of change

Documentation

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by ines** [comment]
Ahh, I hadn't thought of that. It was supposed to be a reference to the "If you need to match large terminology lists..." part in the sentence above, but I can see how the naming is unideal if you just look at the standalone example.

I still think `terminologies` is a bit confusing on its own. How about just `terms`?

File: website/docs/usage/rule-based-matching.md [hunk 1]
```diff
@@ -712,9 +712,9 @@ from spacy.matcher import PhraseMatcher
 
 nlp = spacy.load('en_core_web_sm')
 matcher = PhraseMatcher(nlp.vocab)
-terminology_list = [u"Barack Obama", u"Angela Merkel", u"Washington, D.C."]
+terminologies = [u"Barack Obama", u"Angela Merkel", u"Washington, D.C."]
```
Comment (ines) [inline website/docs/usage/rule-based-matching.md]:
```suggestion
terms = [u"Barack Obama", u"Angela Merkel", u"Washington, D.C."]
```

File: website/docs/usage/rule-based-matching.md [hunk 2]
```diff
@@ -712,9 +712,9 @@ from spacy.matcher import PhraseMatcher
...
 # Only run nlp.make_doc to speed things up
-patterns = [nlp.make_doc(text) for text in terminology_list]
+patterns = [nlp.make_doc(text) for text in terminologies]
```
Comment (ines) [inline website/docs/usage/rule-based-matching.md]:
```suggestion
patterns = [nlp.make_doc(text) for text in terms]
```

**Comment by pickfire** [comment]
@ines Nice findings. :+1:

**Pull request merged**